lib/offline_sort/offline_sort.rb

```diff
@@ -11,67 +12,44 @@ class Sorter
     DEFAULT_CHUNK_IO_CLASS = defined?(::MessagePack) ? Chunk::InputOutput::MessagePack : Chunk::InputOutput::Marshal
     DEFAULT_CHUNK_SIZE = 1000
 
-    attr_reader :enumerable, :sort_by, :chunk_size, :chunk_input_output_class
+    attr_reader :enumerable, :sort_by, :chunk_size, :chunk_input_output_class, :sorted_chunks
 
     def initialize(enumerable, chunk_input_output_class: DEFAULT_CHUNK_IO_CLASS, chunk_size: DEFAULT_CHUNK_SIZE, &sort_by)
       @enumerable = enumerable
       @chunk_input_output_class = chunk_input_output_class
       @chunk_size = chunk_size
       @sort_by = sort_by
+      @sorted_chunks = []
```

Comment:
Does `sorted_chunks` still need to be captured in an instance variable here? Or is it fine for the reference to this array to be maintained in the new `Merger` class?